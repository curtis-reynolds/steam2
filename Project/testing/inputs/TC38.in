login
admin
refund
invalid_buyer_username
invalid_seller_username
