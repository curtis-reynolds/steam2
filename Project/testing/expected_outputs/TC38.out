Login successful.
Error: Buyer or seller does not exist.
